tests/e2e/support/utils.js

```diff
@@ -243,3 +243,13 @@ export function assertDownloadedXMLMatchesExpected(xmlString) {
       expect(xml).to.contain(removeIndentationAndLinebreaks(xmlString));
     });
 }
+
+export function assertDownloadedXmlDoesNotContainExpected(xmlString) {
+  cy.get('[data-test=downloadXMLBtn]').click();
+  cy.window()
+    .its('xml')
+    .then(removeIndentationAndLinebreaks)
+    .then(xml => {
+      expect(xml).to.not.contain(removeIndentationAndLinebreaks(xmlString));
+    });
+}
```

Comment:
This method is the exact same as the one above it, with the simple addition of a `.not` to the assertion. Not sure if it's worth reducing the duplication for this.